src/redux/books.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-unused-vars */
```

Comment:
[Optional] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.